From 257dcf2f80b8620cff384ecfc2b877b9d8ddc7ad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mos=C3=A8=20Giordano?= <mose@gnu.org>
Date: Sat, 4 Jan 2025 01:48:23 +0100
Subject: [PATCH] GCC 14: Increase fallback min macOS version for libgcc to
 10.8

`-mmacosx-version-min=10.5` seems to cause problems with ld64 from cctools
(https://github.com/JuliaPackaging/Yggdrasil/pull/10132#issuecomment-2569925646),
and in any case in Julia/BinaryBuilder we support macOS 10.12 at minimum.
---
 libgcc/config.host | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libgcc/config.host b/libgcc/config.host
index ee10e99b0a6..58a9930a5da 100644
--- a/libgcc/config.host
+++ b/libgcc/config.host
@@ -243,10 +243,10 @@ case ${host} in
       tmake_file="t-darwin-min-8 $tmake_file"
       ;;
     *-*-darwin1[67]]*)
-      tmake_file="t-darwin-min-5 $tmake_file"
+      tmake_file="t-darwin-min-8 $tmake_file"
       ;;
     *-*-darwin9* | *-*-darwin1[0-5]*)
-      tmake_file="t-darwin-min-5 t-darwin-libgccs1 $tmake_file"
+      tmake_file="t-darwin-min-8 t-darwin-libgccs1 $tmake_file"
       ;;
     *-*-darwin[4-8]*)
       tmake_file="t-darwin-min-1 t-darwin-libgccs1 $tmake_file"
@@ -254,8 +254,8 @@ case ${host} in
     *)
       # Fall back to configuring for the oldest system known to work with
       # all archs and the current sources.
-      tmake_file="t-darwin-min-5 t-darwin-libgccs1 $tmake_file"
-      echo "Warning: libgcc configured to support macOS 10.5" 1>&2
+      tmake_file="t-darwin-min-8 t-darwin-libgccs1 $tmake_file"
+      echo "Warning: libgcc configured to support macOS 10.8" 1>&2
       ;;
   esac
   # We are not using libtool to build the libs here, so we need to replicate
